# -*- coding: utf-8 -*-
# Generated by Django 1.9.4 on 2016-06-28 17:02
from __future__ import unicode_literals

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('datastats', '0026_auto_20160628_1701'),
    ]

    operations = [
        migrations.CreateModel(
            name='ResultGeneralAPKInstError',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('guid', models.CharField(max_length=100)),
                ('op', models.CharField(max_length=20)),
                ('version', models.CharField(max_length=50)),
                ('errcode', models.CharField(max_length=20)),
                ('datetime', models.DateTimeField(verbose_name='GeneralData uptime\u5b57\u6bb5')),
            ],
            options={
                'db_table': 'result_general_apk_inst_error',
            },
        ),
        migrations.CreateModel(
            name='ResultGeneralEngineInstall',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('guid', models.CharField(max_length=100)),
                ('op', models.CharField(max_length=20)),
                ('version', models.CharField(max_length=50)),
                ('engine_type', models.CharField(max_length=20)),
                ('datetime', models.DateTimeField(verbose_name='GeneralData uptime\u5b57\u6bb5')),
            ],
            options={
                'db_table': 'result_general_engine_install',
            },
        ),
        migrations.CreateModel(
            name='ResultGeneralEngineInstError',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('guid', models.CharField(max_length=100)),
                ('op', models.CharField(max_length=20)),
                ('version', models.CharField(max_length=50)),
                ('errcode', models.CharField(max_length=20)),
                ('datetime', models.DateTimeField(verbose_name='GeneralData uptime\u5b57\u6bb5')),
            ],
            options={
                'db_table': 'result_general_engine_inst_error',
            },
        ),
        migrations.CreateModel(
            name='ResultGeneralEngInitError',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('guid', models.CharField(max_length=100)),
                ('op', models.CharField(max_length=20)),
                ('version', models.CharField(max_length=50)),
                ('errcode', models.CharField(max_length=20)),
                ('datetime', models.DateTimeField(verbose_name='GeneralData uptime\u5b57\u6bb5')),
            ],
            options={
                'db_table': 'result_general_eng_init_error',
            },
        ),
        migrations.CreateModel(
            name='ResultGeneralOsVersion',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('guid', models.CharField(max_length=100)),
                ('op', models.CharField(max_length=20)),
                ('version', models.CharField(max_length=50)),
                ('status', models.CharField(default='success', max_length=20)),
                ('osver', models.CharField(max_length=20)),
                ('memory', models.CharField(max_length=20)),
                ('cpu', models.CharField(max_length=128)),
                ('datetime', models.DateTimeField(verbose_name='GeneralData uptime\u5b57\u6bb5')),
            ],
            options={
                'db_table': 'result_general_osver',
            },
        ),
    ]
